File: IntergalacticeAirways.DataAccess/Model/Starship.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IntergalacticeAirways.DataAccess.Model
{
    public class Starship
    {
        public string Name { get; set; }
        public string Model { get; set; }
        public string Manufacturer { get; set; }
        public string Passengers { get; set; }
        public IEnumerable<string> Pilots { get; set; }
    }
}

File: UI/IntergalacticAirwaysUI/src/app/model/starship.ts
export class Starship {

    // {
    //     "name": "Trade Federation cruiser",
    //     "model": "Providence-class carrier/destroyer",
    //     "manufacturer": "Rendili StarDrive, Free Dac Volunteers Engineering corps.",
    //     "passengers": "48247",
    //     "pilots": [
    //       "https://swapi.dev/api/people/10/",
    //       "https://swapi.dev/api/people/11/"
    //     ]
    //   },
    name: string | undefined;
    model: string | undefined;
    manufacturer: string| undefined;
    passengers: string| undefined;
    pilots: string[] | undefined;
}
